src/GeneralStateTestsFiller/stTransactionTest/Opcodes_TransactionInitFiller.json

```diff
@@ -323,7 +348,8 @@
                 ":raw 0x6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6612710fa5060006000f3", 
                 ":raw 0x60006000fd60006000f3", 
                 ":raw 0x32ff",
-                ":label invalid_first_byte_ef :raw 0xef"
+                ":label invalid_first_byte_ef :raw 0xef",
+                ":label side_effects_invalid_opcode :yul { pop(call(50000, 0xb94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0, 0)) invalid() }"
```

Comment:
invalid is like revert?

Reply:
here is tricky to know that the call worked. 
if its not requred to call revert directly in transaction init data field. 
the way to do it is to go subcall. 

transaction init ->   call ->  do smth.  revert.  |  continue execution. sstore smth. 

this way revert should cause only a subcall going oog and we can confirtm that execution happend by putting a sstore at the end of transaction init code.

Reply:
or you can have a control transaction with exactly the same data,  but this time without invalid() instruction at the end. 
so we check that the call (50000, ...  worked as at some point 50000 might be not enough and it will just go OOG not testing what we wanted

Reply:
I don't want to test here reverts inside subcalls, becasue this file is all about init code in the creation transaction.
(Although other similar tests to check reverts inside CREATEs might be useful.)

So I prefer to add another similar transaction with the same side effect, but without revert.

Reply:
Added.